tests/v2/validation/provisioning/ssh.go

```diff
@@ -0,0 +1,39 @@
+package provisioning
+
+// This file contains all tests that require to ssh into a node to run commands to check things
+// such as any stats, benchmarks, etc. For example, ssh is required to check the cpu usage of a
+// process running on an individual node.
+
+import (
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	cpuUsageVar = 100
```

Comment:
how was this number chosen?

Reply:
I just set this value as a placeholder. I can change this value. When users were hitting this issue in the past the usage was around 130-170%. I just set it to 100% because even if the cpu were to spike for some reason it shouldn't go past 100%. Thoughts?

Reply:
that makes sense to me Tim. I agree it shouldn't go above 100%, so 100 seems like an ideal value here IMO

Reply:
some (maybe all) systems show a single process' usage where 100% == 1 core. that's why 100 seemed 'arbitrary' for lack of a better word. But if its based on previous tests, may be worth a comment in the code on why. Otherwise, without that context it does read as 'if your cpu is running at more than full throttle'. But since this is just for a `warn` log it should be fine.

Reply:
Thinking on Caleb's comment above, leaving a comment would be good. Just to avoid future confusion as to how this number was chosen.